### ISSUE TITLE
Suggestion: Borderless Mode

### ISSUE THREAD

**Issue by themegaxandy** [Naezr/ShyFox#140]
Currently, when using the existing clean mode while the window is maximized, the borders are hidden, while it is still possible to drag the window by an invisible border, which is great.
It would be nice if the same behavior occurred in window mode, so that more space could be gained in the smaller window. Clean mode is great for windows where there is only one tab, and the user does not need to use any of the border tools at that time.

Another related suggestion that came to mind is about the F11 fullscreen mode. I loved that this mode allows you to access the border tools even though they are visually hidden. However, with F11 you don't have easy access to the Windows taskbar, as it is hidden.
It would be interesting if this same mode were applied to maximized windows, providing a borderless UI but maintaining access to the tools and dragging of windows. In addition, the Windows taskbar would be visible.

I know that not everyone may like these changes, so a toggle could be created for this, taking advantage of the ShyFox variables available in about:config. One idea of ​​how these modifications could be called could be "Borderless mode", where the user prefers not to have visible borders, but still have access to the border tools, and can even fix them visible as it happens in the full screen mode F11.

**Comment by Naezr** [comment]
I hadn't really thought about such usage scenarios, since I use Linux with the GNOME desktop environment. There is no taskbar there, just a panel on top, roughly like macOS. 

It's an interesting idea, something like that will definitely come up in the future

**Comment by KAGEYAM4** [comment]
Yes it would be great if this could be possible in future. Something like this ->

![image](https://github.com/user-attachments/assets/b3ec059a-83c6-4246-8cb0-5be0edf62dcd)
